docs/school-check-data.md

```diff
@@ -52,10 +52,15 @@ which need this email sending is a developer task, documented below.
    `school-check-batch_ddmmyyyy.csv`
 
 8. Securely transfer the file to the relevant service operator to follow the
-   rest of the school check process
+   rest of the school check process. The best way we’ve found to do this is to
+   create a new Google Sheets document, private to yourself and shared only with
```

Comment:
```suggestion
   create a [new Google Sheets document](https://sheets.new/), private to yourself and shared only with
```

Reply:
Is this documented by Google somewhere?

Reply:
Found this: https://twitter.com/googledocs/status/1055490445088903168

Reply:
https://support.google.com/a/users/answer/9308871?hl=en has an overview.